src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+
+public class HelloWorld {
+    public static void main(String[] args) {
+
+    }
+
+    private String getFromOneToTen() {
+        String result = "";
+
+        for (int i = 0; i < 10; i++) {
+
+            if (i % 2 == 0) {
+                result += i + " ";
+            }
+
+        }
+        return result;
```

Comment:
```suggestion
        String result = "";
        for (int i = 0; i < 10; i++) {
            if (i % 2 == 0) {
                result += i + " ";
            }
        }
        return result;
```
Let’s  remove  redundant empty lines